docs/HowToSetEnv.md

```diff
@@ -6,44 +6,82 @@ Get the build in binaries and libraries, [gcc-arm-none-eabi-6-2017-q1-update-*OS
 Untar the gcc-arm-none-eabi-6-2017-q1-update-*OS Type*.tar.bz2 and export the path like
 
 ```bash
-tar xvjf gcc-arm-none-eabi-6-2017-q1-update-[OS Type].tar.bz2
-export PATH=<Your Toolchain PATH>:$PATH
+	$  tar xvjf gcc-arm-none-eabi-6-2017-q1-update-[OS Type].tar.bz2
+	$  export PATH=<Your Toolchain PATH>:$PATH
 ```
-Be aware that recommended toolchain is fully working on 64bits machine.
+Be aware that the recommended toolchain is fully working on 64bits machine.
 
 ## Getting the source code
+To get the source code git clone to your local machine, move to the TizenRT directory and setup an environment variable
+for your TizenRT base directory:
 
 ```bash
-git clone https://github.com/Samsung/TizenRT.git
-cd TizenRT
-TIZENRT_BASEDIR="$PWD"
+	$  git clone https://github.com/Samsung/TizenRT.git
+	$  cd TizenRT
+	$  TIZENRT_BASEDIR="$PWD"
 ```
 
 ## How to Build
+To build TizenRT binaries there are 2 steps: 
+1) configuring your build for a particular board and configuration set; and 
+2) performing the build.
 
-Configure the build from *$TIZENRT_BASEDIR/os/tools* directory
+Before you do the configure you will need to know what board and configuration set you would like to use.
```

Comment:
Decide the board and configuration set you want to build and configure accordingly. To know the list of available boards and configuration sets, run the configure script with help option as below.